Userland/Libraries/LibCore/Notifier.h

```diff
@@ -11,8 +11,8 @@
 
 namespace Core {
 
-class Notifier : public Object {
-    C_OBJECT(Notifier)
+class AbstractNotifier : public Object {
+    C_OBJECT_ABSTRACT(AbstractNotifier)
```

Comment:
Let's put an `;` at the end of the line for consistency with the rest of the codebase.